src/main.js

```diff
@@ -7,12 +7,11 @@ const path = require('path');
 const url = require('url');
 const fixPath = require('fix-path');
 const chalkRaw = require('chalk');
-const ElectronStore = require('electron-store');
 
 const killProcessId = require('./services/kill-process-id.service');
+const electronStore = require('./services/electron-store.service');
```

Comment:
I moved all of our electron-store calls to a service so that we could expose a singleton. This makes testing easier (since that singleton is mocked consistently), as well as just feels more intuitive (it was weird that different instances had access to the same underlying storage)